### PR TITLE
Fixed broken syntax (change <> to !=)

### PR DESCRIPTION
**Pull request by scovetta**
I have not tested this, just came across it and it looks like a trivial fix.

### CODE REVIEW

File: gpMgmt/bin/gppylib/datetimeutils.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def scan_datetime(string, pos=0, endpos=None):
         yyyy, m, d, mm, dd = datematch.groups()
         if m:
             # delimited format
-            if len(d) <> 2 and len(d) <> 1:
+            if len(d) != 2 and len(d) != 1:
```
Comment (danielgustafsson) [inline gpMgmt/bin/gppylib/datetimeutils.py]: Yes and no, it's not actually broken. From the [2.7 documentation](https://docs.python.org/release/2.7/library/stdtypes.html#comparisons):
> != can also be written <>, but this is an obsolete usage kept for backwards compatibility only. New code should always use !=.

This code was written for Python <= 2.7, thus the use of old crusty operators. That being said, `<>` is removed from Python 3 so there is no reason to not change as per this patch.

**Comment by danielgustafsson** [comment]
@jchampio unless you (or anyone else) have objections I will go ahead and merge this tomorrow after a CI run

**Comment by jchampio** [comment]
@danielgustafsson Thanks! Seems good from manual inspection, though I'm not sure how much coverage we get from the PR pipeline here.

**Comment by danielgustafsson** [comment]
> On 12 Feb 2020, at 22:31, Jacob Champion <notifications@github.com> wrote:
> 
> @danielgustafsson Thanks! Seems good from manual inspection, though I'm not sure how much coverage we get from the PR pipeline here.

Probably nothing (not volunteering), but I like consistency in process and to keep PRs open for one pass across the timezones.

**Comment by jchampio** [comment]
@scovetta Out of curiosity, what motivated this particular fixup?

**Comment by scovetta** [comment]
@jchampio I run an open source security team at Microsoft, and we use [CodeQL](https://lgtm.com) to analyze open source. We usually focus on security issues, but this one caught my eye and figured it'd be an easy PR. I wasn't aware that '<>' was permitted in older versions. :-)

**Comment by jchampio** [comment]
@scovetta Cool. Thanks for the fix!

**Comment by danielgustafsson** [comment]
Pushed to master with a slightly tweaked commit message, thanks for your contribution!

**Pull request closed without merging**